packages/pf4-component-mapper/src/files/dual-list-select.js

```diff
@@ -306,7 +332,7 @@ DualList.defaultProps = {
   filterValueText: 'Remove your filter to see all selected',
   options: [],
   allToLeft: true,
-  allToRight: true
+  allToRight: true,
```

Comment:
I think linter will fail here (trailing comma)

Reply:
dang - you're right!

Reply:
fixed :man_dancing: